### ISSUE TITLE
ReferenceError: jQuery is not defined – vendor.min.js:1:872

### ISSUE THREAD

**Issue by adrian5** [getgrav/grav-plugin-admin#1140]
Since it's very difficult to get Grav related help, I thought I'd ask here, even if the problem is likely on me, rather than a bug.

I noticed that the Admin Plugin doesn't allow me to scroll anymore and that a few Dashboard widgets are stuck loading. Opening the console, I found above message about jQuery.

I recently added new assets to `/user/config/system.yaml`, so it might have something to do with that. I now have it looking like this again:

```
assets:
  css_pipeline: false
  css_pipeline_include_externals: false
  css_pipeline_before_excludes: false
  css_minify: false
  css_minify_windows: false
  css_rewrite: false
  js_pipeline: true
  js_pipeline_include_externals: false
  js_pipeline_before_excludes: false
  js_minify: false
  enable_asset_timestamp: true
  collections:
    jquery: system://assets/jquery/jquery-2.1.3.min.js
```

Any suggestions welcome.

**Comment by adrian5** [comment]
Ah nevermind, it's solved. I took the jQuery string from [this article](https://learn.getgrav.org/themes/asset-manager#named-assets), but Grav ships with jQuery 2.1.**4** by now. After changing the digit, the Admin Plugin works again.

**Comment by flaviocopes** [comment]
We have solved this issue a while ago by adding `assets/jquery/jquery-2.x.min.js` and `assets/jquery/jquery-3.x.min.js` which always point to the latest release of jQuery of the corresponding major release. You don't need to specify the exact minor release number: https://github.com/getgrav/grav/tree/develop/system/assets/jquery

The problem was that when updating, to avoid breaking the user configuration which was pointing at the old release (like in your case), we needed to ship the old jQuery releases in Grav, and over time this would have been a nightmare.

Good you solved, but should not happen with newer releases of Grav. If you instead have just installed it, on the other hand, this is not supposed to happen.
